### ISSUE TITLE
train.py

### ISSUE THREAD

**Issue by jenwashco** [jenwashco/DSND-Project-2#1]
%matplotlib inline
%config InlineBackend.figure_format = 'retina'

import argparse
import matplotlib.pyplot as plt
import torch
import numpy as np
from torch import nn
from torch import optim
import torch.nn.functional as F
from torchvision import datasets, transforms, models
import tensorflow as tf
from collections import OrderedDict

# Create parser for command line input
parser = argparse.ArgumentParser(description = 'train.py parser')

parser.add_argument('data_dir', type=str, help='Path to dataset')
parser.add_argument('--save_dir', type=str, help='Path to save directory')
parser.add_argument('--arch', type=str, help='Model architecture')
parser.add_argument('--learning_rate', type=float, help='Learning rate')
parser.add_argument('--hidden_units', type=int, help='Number of hidden units')
parser.add_argument('--epochs', type=int, help='Number of epochs')
parser.add_argument('--gpu', action='store_true', help='Use GPU when possible')

# this passes addl args to other scripts and programs as needed
args, _ = parser.parse_known_args()

# Load model according to specifications
def load_model():
    # if statements to use cmd line args: arch, data_dir
    
    # load specified model
    checkpoint = torch.load(filepath, map_location=lambda storage, location: storage)
    model = models.vgg16(pretrained=True)
    model.classifier = checkpoint['classifier']
    model.class_to_idx = checkpoint['class_to_idx']
    model.load_state_dict(checkpoint['state_dict'])
    
    return model

# Train model according to specifications
def train_model():
    
    return model

# Train model when specified in command line
if args.data_dir:
    # fill in
    
load_model()
